system/netdata.logrotate.in

```diff
@@ -0,0 +1,15 @@
+@localstatedir_POST@/log/netdata/*.log {
+	daily
+	missingok
+	rotate 14
+	compress
+	delaycompress
+	notifempty
+	sharedscripts
+	postrotate
+		if service netdata status > /dev/null ; then \
+			service netdata restart > /dev/null; \
+		fi;
```

Comment:
> The best solution would be to signal to the daemon to reopen its log files (-HUP, -USR1, etc). Admit I haven't looked to see if it will currently do this, though.

Hm... I don't know if this can be done. The whole process tree of netdata has `stderr` attached to `error.log`. Most children (bash, node.js, anything) are not aware of this. So, even if the daemon re-opens its files, the children will not.